resources/META-INF/plugin.xml

```diff
@@ -334,6 +334,8 @@
     <action id="VimSearchWordBackward" class="com.maddyhome.idea.vim.action.motion.search.SearchWordBackwardAction" text="Search Again"/>
     <action id="VimGotoDeclaration" class="com.maddyhome.idea.vim.action.motion.search.GotoDeclarationAction" text="Goto Declaration"/>
 
+    <action id="VimSelectNextOccureance" class="com.maddyhome.idea.vim.action.motion.visual.VisualSelectNextOccurrence" text="Select Next Occurrence"/>
```

Comment:
Should be `VimSelectNextOccurrence`

Reply:
Thank you for contributing and valuable remark!